### PR TITLE
Datagrid handles visible

### PR DESCRIPTION
**Pull request by karthik-sista**

### CODE REVIEW

**Comment by karthik-sista** [comment]
That was just a wip, just confirmed with matt and working on it.

**Comment by karthik-sista** [comment]
data-grid-handles

**Comment by karthik-sista** [comment]
Screen Shot 2019-05-06 at 10 36 10 AM

**After adding spacing on top and bottom**

File: libs/datagrid/src/DataGrid.module.scss [hunk 1]
```diff
@@ -81,8 +81,11 @@
     }
     .rt-resizer {
       width: 4px;
-      right: -2px;
+      right: 0px;
       z-index: 0;
+      border-right: 1px solid #b3b3b3;
```
Comment (eckdanny-osi) [inline libs/datagrid/src/DataGrid.module.scss]: `$variables` plz

**Pull request merged**